packages/vc-http-api-test-server/__tests__/issueCredential.spec.js

```diff
@@ -120,6 +120,49 @@ if (suiteConfig.issueCredentialConfiguration) {
           expect(res.status).toBe(201);
           expect(res.body.proof).toBeDefined();
         });
+
+        if (value.credentialStatusesSupported) {
+          const credentialStatusesSupported = Array.isArray(value.credentialStatusesSupported) ? value.credentialStatusesSupported : [ value.credentialStatusesSupported ];
+
+          credentialStatusesSupported.forEach((credStatusType) => {
+            it(`10. The Issuer's Issue Credential HTTP API MAY support issuing a credential with a credential status type of ${credStatusType}`, async () => {
```

Comment:
Numbering each test seems strange... may want to rely on mocha reporting mechanism to do that.

Reply:
@msporny the reason these are numbered is because we needed a way to talk about the tests for issuer, holder and verifier without asking produce folks to look at code.

I agree this is odd, but don't have a better solution.